boards/arm/dragonclaw/dragonclaw.yaml

```diff
@@ -0,0 +1,14 @@
+identifier: dragonclaw
+name: Google Dragonclaw Development Board
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 256
+flash: 1024
+testing:
+  ignore_tags:
+    - net
+    - bluetooth
```

Comment:
I don't think these are needed, there's not ble or network device in the dts that could trigger these tests anyway.